bin/pygrb/pycbc_pygrb_plot_coh_ifosnr

```diff
@@ -233,10 +235,10 @@ ifos, vetoes = ppu.extract_ifos_and_vetoes(
 )
 
 # Extract trigger data
-trig_data = load_data(trig_file, ifos, vetoes, opts)
+trig_data = load_data(trig_file, ifos, vetoes, opts, slide_id=0)
 
 # Extract (or initialize) injection data
-inj_data = load_data(found_file, ifos, vetoes, opts, injections=True)
+inj_data = load_data(found_file, ifos, vetoes, opts, injections=True, slide_id=0)
```

Comment:
I would add a `slide_id` option to plotting scripts like this one, with 0 as default, and explaining that `None` will force plotting all triggers from all timeslides; then I would pass this to `load_data` via the keyword `slide_id`

Injections are on unslid data so `inj_data = load_data(found_file, ifos, vetoes, opts, injections=True, slide_id=0)` can be left unchanged.

Reply:
I just finished implementing this, let me know whether I misunderstood your request.